frontend/src/reducer/workspaceContentList.js

```diff
@@ -32,6 +32,7 @@ export const serializeContentProps = {
   is_template: 'isTemplate',
   isOpen: 'isOpen',
   label: 'label',
+  last_modifier: 'lastModifier',
```

Comment:
I think lastModifier is an object, therefore, ideally, that object is also serialized when put in redux

Reply:
it's something we haven't done everywhere in the past and it led to some boolean expression looking like
```jsx
if (props.something_about_something === true || props.somethingAboutSomething === true) {
```
which we must be avoided